src/vs/editor/contrib/quickFix/quickFixModel.ts

```diff
@@ -36,12 +37,12 @@ export class QuickFixOracle {
 		this._disposables = dispose(this._disposables);
 	}
 
-	trigger(type: 'manual' | 'auto'): void {
+	trigger(type: 'manual' | 'auto', trigger?: CodeActionTrigger): void {
```

Comment:
getting a little messy... can we fold both args into one request object and pass that along?